### ISSUE TITLE
Program crashes when trying to open existing config file

### ISSUE THREAD

**Issue by JonathanStreet** [unobserved-io/espanso-gui#1]
## Steps to reproduce the bug:

* Create a new config file `crash.yml` under the default espanso configuration location (`%appdata%\espanso\match` on Windows) containing the following:
```
- regex: "input"
  replace: "output"
```
* Open EspansoGUI, and try to open the file.

## Resulting and expected behaviour:

Program crashes (no errors).
Expect the file to be loaded with no contents (as there's nothing in the file matching 'trigger/replace' in the yaml tree.

## System/Device Information:

* **Operating System:** Windows 10
* **EspansoGUI Build:** 2023.09.15 - espansoGUI-2023.09.15-x86_64.msi

**Comment by JonathanStreet** [comment]
Was also able to recreate with the following - however, this one isn't a valid espanso config!

```
- trigger: "input"
  crash: "output"
```

**Comment by rickykresslein** [comment]
Thanks for pointing this out! Right now it only expects trigger/replace, so I will work on updating it to not crash when something else is loaded.

**Comment by rickykresslein** [comment]
This bug is now fixed with e6937fd. espansoGUI cannot read the regex line as of now, and that line will be erased if you save a file that used it, but it will no longer crash the program. I've added regex to the list of features to add.
